package.json

```diff
@@ -40,7 +40,7 @@
     "react": "16.14.0",
     "react-bootstrap": "^1.0.0-beta.6",
     "react-dom": "^16.14.0",
-    "react-mutation-mapper": "^0.8.9",
+    "react-mutation-mapper": "^0.8.10",
```

Comment:
@leexgh we should update react-mutation-mapper to the latest version to get the table controls style changes, right?

Reply:
@onursumer yes I was about to update. Also updated cbioportal-frontend-commons